staging/src/kubevirt.io/client-go/api/BUILD.bazel

```diff
@@ -3,6 +3,6 @@ load("@io_bazel_rules_go//go:def.bzl", "go_library")
 go_library(
     name = "go_default_library",
     srcs = ["types.go"],
-    importpath = "kubevirt.io/kubevirt/pkg/api",
+    importpath = "kubevirt.io/kubevirt/staging/src/kubevirt.io/client-go/api",
```

Comment:
@rmohr By the way because of it, the building will fail if we will use the symbolic link, we use imports of the form `kubevirt.io/kubevirt/client-go/api` but `importpath` is `kubevirt.io/kubevirt/staging/src/kubevirt.io/client-go/api`, need to check if gazelle has the option to specify the `importpath` for the whole package or overwrite it with `buildozier`